src/tiles/grid/collection-tile.ts

```diff
@@ -21,18 +21,20 @@ export class CollectionTile extends LitElement {
         }}
```

Comment:
Let's remove these `@mouseover` and `@focus` listeners.. I think I put them in trying to debug that Safari issue.